src/Eurofurence.App.Domain.Model/ArtistsAlley/UserStatus/ArtistAlleyUserPenaltyRecord.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Eurofurence.App.Domain.Model.ArtistsAlley
+{
+
+    
+    /// <summary>
+    /// Record for storing
+    /// </summary>
+    [Table("ArtistAlleyUserPenalties")]
+    public class ArtistAlleyUserPenaltyRecord : EntityBase
+    {
+
+        public ArtistAlleyUserPenaltyRecord()
+        {
+            Penalty = UserPenalties.OK;
+        }
+
+        /// <summary>
+        /// Possible penalties of a user
+        /// </summary>
+        public enum UserPenalties
+        {
+            BANNED = 0,
+            OK = 1
+        }
+
+        /// <summary>
+        /// Id of the user from the reg system
+        /// </summary>
+        [Column("UserId")]
+        public string UserId { get; set; }
```

Comment:
Not sure if we want the ID from the registration system or rather the ID from the identity provider here, but the naming should be standardised to either

RegSysId
or
IdentityId

to make it more clear which ID to expect in this field.